src/backend/main/py.main/caliopen_main/message/core/raw.py

```diff
@@ -22,19 +23,19 @@ class RawMessage(BaseUserCore):
     """
 
     _model_class = ModelRaw
-    _pkey_name = 'raw_id'
+    _pkey_name = 'raw_msg_id'
 
     @classmethod
-    def create(cls, user, message_id, raw):
+    def create(cls, user, raw):
         """Create raw message."""
-        key = hashlib.sha256(message_id).hexdigest()
-        return super(RawMessage, cls).create(user, raw_id=key, data=raw)
+        key = uuid.uuid4()
```

Comment:
uuid.uuid4() still in use in : 

* core/contact
* core/thread
* core/message

you cleanup or I cleanup ?